deployment/cdk/opensearch-service-migration/lib/network-stack.ts

```diff
@@ -45,6 +45,20 @@ export class NetworkStack extends Stack {
         return formattedUrlString
     }
 
+    private validateVPC(vpc: IVpc) {
+        let uniqueAzPrivateSubnets: string[] = []
+        if (vpc.privateSubnets.length > 0) {
+            uniqueAzPrivateSubnets = vpc.selectSubnets({
+                subnetType: SubnetType.PRIVATE_WITH_EGRESS,
+                onePerAz: true
+            }).subnetIds
+        }
+        console.info(`Detected VPC with ${vpc.privateSubnets.length} private subnets, ${vpc.publicSubnets.length} public subnets, and ${vpc.isolatedSubnets.length} isolated subnets`)
+        if (uniqueAzPrivateSubnets.length < 2) {
+            throw new Error('Not enough AZs used for private subnets to meet 2 or 3 AZ requirement')
```

Comment:
Include uniqueAzPrivateSubnets.length in the log since it wasn't included in the output above.

Reply:
Added